### PR TITLE
Do not check services endpoint when running jobs

### PR DESCRIPTION
**Pull request by ayinloya**
We currently check the dynamo hosted web collection to see id types and countries supported before summiting jobs to the API.
The check is no longer necessary.

### CODE REVIEW

File: smile_id_core/Utilities.py [hunk 1]
```diff
@@ -247,50 +247,6 @@ def validate_id_params(
                 raise ValueError(f"key {field} cannot be empty")
             else:
                 raise ValueError(f"key {field} cannot be empty")
-        if not use_validation_api:
```
Comment (vanshg) [inline smile_id_core/Utilities.py]: We should also remove `use_validation_api` from the function signature (and therefore update the changelog and bump the version?)

Comment (ayinloya) [reply smile_id_core/Utilities.py]: We can deprecate it for now and remove it later in a later version

Comment (ayinloya) [reply smile_id_core/Utilities.py]: You're right, we should remove it entirely

**Review by beaglebets — APPROVED** [review]

**Review by mldangelo — CHANGES_REQUESTED** [review]
Please update the change log with an unreleased entry

**Comment by ayinloya** [comment]
> Please update the change log with an unreleased entry

Done

**Review by mldangelo — APPROVED** [review]
Thanks!

**Pull request merged**